cmake/Jucer2CMake/main.cpp

```diff
@@ -36,36 +36,16 @@ void printError(const std::string &error) {
 
 String makeValidIdentifier(String s) {
   jassert(!s.isEmpty());
-
-  s = s.replaceCharacters(".,;:/@", "______");
-
-  for (int i = s.length(); i > 0; --i) {
-    if (CharacterFunctions::isLetter(s[i]) &&
-        CharacterFunctions::isLetter(s[i - 1]) &&
-        CharacterFunctions::isUpperCase(s[i]) &&
-        !CharacterFunctions::isUpperCase(s[i - 1])) {
-      s = s.substring(0, i) + " " + s.substring(i);
+  auto stdString = s.toStdString();
```

Comment:
@MartyLake: there are too many back-and-forth between `juce::String` and `std::string` for my taste. Let's try to use `juce::String` as less as possible.